build/index.js

```diff
@@ -155,9 +155,13 @@ function executeProcess(executable, args, env = process.env) {
         });
     });
 }
-
-const npm = (...args) =>
-    executeProcess("npm", args, { ...process.env, FORCE_COLOR: 1 });
+const npm = async (...args) => {
+    const start = new Date();
+    const cmd = windowsEnvironment ? "npm.cmd" : "npm";
+    await executeProcess(cmd, args, { ...process.env, FORCE_COLOR: 1 });
+    const elapsed = new Date() - start;
+    logDetail(`NPM finished in ${elapsed} ms`);
```

Comment:
I also noticed that this seems to be pulling in some changes that have already landed on `main` - can't entirely explain it, but it's odd.

Reply:
Probably a merge without rebase.